Sources/Extensions/Foundation/URLExtensions.swift

```diff
@@ -63,6 +63,20 @@ public extension URL {
 		self = appendingQueryParameters(parameters)
 	}
 
+    /// SwifterSwift: Get value of a query key.
+    ///
+    ///    var url = URL(string: "https://google.com?code=12345")!
+    ///    queryValue(for: "code") -> "12345"
+    ///
+    /// - Parameter: Key of a query.
```

Comment:
This should be:
`/// - Parameter key: The key of a query value.`

Reply:
Good catch.